pkg/pfcpsim/session/far_builder_test.go

```diff
@@ -0,0 +1,34 @@
+package session
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFARBuilderShouldPanic(t *testing.T) {
+	assert.Panics(t, func() {
```

Comment:
Didn't find a proper way to setup a table testing scenario while testing panics

Reply:
maybe I could do something like this: https://gist.github.com/Nitecon/3e10e05af19b13f21e77a8160f138c58 But I don't like it much